hocs/securePage.js

```diff
@@ -24,16 +37,27 @@ const securePageHoc = Page =>
       );
     }
 
+    componentDidMount() {
+      setRedirectUrl({
+        pathname: this.props.url.pathname,
+        asPath: this.props.url.asPath
+      });
+    }
+
     render() {
       if (!this.props.isAuthenticated) {
         return (
-          <Layout>
+          <Layout crumbs={[<Trans>Login required</Trans>]}>
             <Container pt={50}>
-              Please{' '}
-              <Link href="/auth/sign-in" prefetch>
-                <a>login</a>
-              </Link>{' '}
-              to continue
+              <Box textAlign="center">
+                <Trans>
```

Comment:
Trenger vi egentlig en egen side som sier `Please login to continue.` og at du da må klikke på lenken for å komme til den egentlige login-siden? Det virker som et unødvendig steg. Hvorfor kan vi ikke komme direkte til siden som har Google og Facebook-knappene?

Reply:
fikset